### PR TITLE
As multi account login

### PR DESCRIPTION
**Pull request by artursmirnov**
Ticket: https://eu-app.assembla.com/spaces/bCSikkvHmr6B5HaMa9TKfk/tickets/275/details

### CODE REVIEW

**Comment by artursmirnov** [comment]
This PR should be ready for review. @sorta @piotr-travisci @emmanuel-ci please take a look when you have a moment. Thanks!

File: app/templates/components/dashboard-row.hbs [hunk 1]
```diff
@@ -145,6 +145,7 @@
                     </button>
                   </li>
                   <li>
+                    {{log this.repo}}
```
[the remark this answers is not in the document]
Comment (artursmirnov) [reply app/templates/components/dashboard-row.hbs]: Yeah, leftover for sure 😅 thanks for catching it 👍

File: app/services/auth.js [hunk 2]
```diff
@@ -99,55 +127,73 @@ export default Service.extend({
...
-    this.autoSignIn();
-    if (this.signedIn) return;
-
+  signIn(provider) {
     this.set('state', STATE.SIGNING_IN);
 
     const url = new URL(this.redirectUrl || window.location.href);
 
-    if (url.pathname === '/plans') {
+    if (['/signin', '/plans'].includes(url.pathname)) {
```
[the remark this answers is not in the document]
Comment (artursmirnov) [reply app/services/auth.js]: Yeah, this url is for after-sign-in redirect. E.g if user starts right on the sign in page, the redirect url would be back to sign in, which doesn't make sense after authorization. So if the user starts on sign in page, we make sure to redirect them to `/` after successful sign in. Although, if user clicks the sign in button on top nav at some other url, it sets `auth.redirectUrl` and redirects them to that url after signing in. Does it make sense?

[the remark this answers is not in the document]
Comment (sorta) [reply app/services/auth.js]: Ahh cool, yeah that makes sense :+1: Thanks for the explanation!

**Comment by artursmirnov** [comment]
Thanks for reviews, everyone! @sorta @emmanuel-ci I suppose it doesn't work on .org because required API changes are not deployed there yet.

**Pull request merged**